packages/manager/apps/web/client/app/hosting/multisite/translations/Messages_fr_FR.json

```
@@ -13,9 +13,12 @@
   "hosting_multisite_statistics_changing_datavis_tool": "L'outil de visualisation des statistiques de votre CDN évolue. Retrouvez désormais vos données <b>sur la page OVHcloud Web Statistics</b> pour vous aider à suivre et piloter le trafic de vos sites web",
   "hosting_multisite_git_association_title": "Configurer votre Git pour votre répertoire {{ path }}",
   "hosting_multisite_git_association_ssh_key": "Clé SSH",
+  "hosting_multisite_git_association_incorrect_url_format": "Le format de l'url n'est pas correct",
   "hosting_multisite_git_association_ssh_url_repo_help": "Indiquer l'URL de votre dépôt Git ({{ httpsRepo }}) ou si SSH est activé ({{ sshRepo }})",
   "hosting_multisite_git_association_ssh_key_internal_error_message": "La génération de votre clé SSH a échoué, veuillez ré-essayer ultérieurement ou contacter le support",
-  "hosting_multisite_git_association_ssh_key_success_message": "Git est en cours d’activation. Vous pouvez suivre son activation depuis les tâches en cours <a href=\"{{ href }}\">Tâches en cours</a>",
+  "hosting_multisite_git_association_apply_configuration_status_error": "Votre dernière configuration a échoué. Afin de configurer Git, vous devez à nouveau appliquer votre configuration.",
+  "hosting_multisite_git_association_error_message": "Git n'a pas pu être activé {{ errorMessage }}",
+  "hosting_multisite_git_association_success_message": "Git est en cours d’activation. Vous pouvez suivre son activation depuis les tâches en cours <a href=\"{{ href }}\">Tâches en cours</a>",
```

```suggestion
  "hosting_multisite_git_association_success_message": "Git est en cours d'activation. Vous pouvez suivre son activation depuis les tâches en cours.",
```
I think we can output the html code `<a href=\"{{ href }}\">Tâches en cours</a>`

---

Nope because it is a message sent to Alerter here